plugins/com.google.cloud.tools.eclipse.appengine.facets/plugin.properties

```diff
@@ -11,3 +11,6 @@ standardFacetDescription = Supply App Engine standard environment functionality
 convertToAppEngineStandardCommandDescription = Converts an Eclipse project into a Google App \
  Engine standard environment project.
 convertToAppEngineStandardCommandName = Convert to App Engine Standard Project
+
+appengine.standard.jre7.preset = App Engine standard environment with Java 7, Servlet 2.5
```

Comment:
Sorry I missed the context. I mean, you used "appengine.standard.java8.preset" in other places, like the preset ID.